src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,59 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class UtilsTest {
+
+	private Object nullObject;
+	private Object twinObjectOne = "twin";
+	private Object twinObjectTwo = "twin";
+	private Object uniqueObject = "unique";
+	private ArrayList<Object> numbers = new ArrayList<Object>();
+	
+	@Test
+	public void isNull(){
```

Comment:
This is not the standard to name the test case. It should be MethodTested_ScenarioTest_returnValue.

i.e., isAnyNull_emptyList_falseReturned